data/data/aws/main.tf

```diff
@@ -77,3 +77,12 @@ module "vpc" {
 
   tags = "${local.tags}"
 }
+
+resource "aws_ami_copy" "main" {
+  name              = "${var.cluster_id}-master"
+  source_ami_id     = "${var.aws_ami}"
+  source_ami_region = "${var.aws_ami_region}"
+  encrypted         = true
+
+  tags = "${local.tags}"
```

Comment:
Can you add the `Name` tag [why](https://github.com/openshift/installer/pull/1280#discussion_r258616827)

Reply:
> Can you add the Name tag why [This allows cluster to find resources using tags based on names][1]

So searching by wildcard?  Does this also apply to resources created by the cluster components (e.g. the machine-API provider)?  I'd expect folks to set this via `userTags` in their `install-config.yaml`.  But if you want an extra tag, I can add it.  Did you just want:

```hcl
tags = "${merge(map(
  "Name", "${var.cluster_id}",
, local.tags)}"
```

[1]: https://github.com/openshift/installer/pull/1280#discussion_r258616827

Reply:
Match the `name`  with `Name`

Reply:
> Match the `name` with `Name`

:man_facepalming: Done with f08e266 -> 4fcef5e.